http/codegen/testdata/result_encode_functions.go

```diff
@@ -442,13 +442,9 @@ var ResultHeaderBoolDefaultEncodeCode = `// EncodeMethodHeaderBoolDefaultRespons
 func EncodeMethodHeaderBoolDefaultResponse(encoder func(context.Context, http.ResponseWriter) goahttp.Encoder) func(context.Context, http.ResponseWriter, interface{}) error {
 	return func(ctx context.Context, w http.ResponseWriter, v interface{}) error {
 		res := v.(*serviceheaderbooldefault.MethodHeaderBoolDefaultResult)
-		if res.H != nil {
-			val := res.H
-			hs := strconv.FormatBool(*val)
-			w.Header().Set("h", hs)
-		} else {
-			w.Header().Set("h", "true")
-		}
+		val := res.H
```

Comment:
The test is not needed anymore?

Reply:
We cannot set default values when sending server response because the attributes with default values are stored as non-pointers in the result type. Hence we won't know if the attribute is set by the user or not.